media-sound/audacity/audacity-3.4.2.ebuild

```diff
@@ -128,6 +128,9 @@ PATCHES=(
 
```

Comment:
Thank you for the review, @cjmayo!

> Bit unfair because it is not the original topic, but I think it should now be:
`mpg123? ( media-sound/mpg123-base )`

I suppose it would be better to handle this in a separate PR.

> and there is [bugs.gentoo.org/927659](https://bugs.gentoo.org/927659) for media-libs/opusfile (haven't checked that but I do have opusfile installed).

The `opus` USE flag turned out to be quite broken.
I've submitted a corresponding PR:
  - #35903